products/storage/inspec.yaml

```diff
@@ -54,7 +54,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     name: BucketObject
     singular_only: true
     privileged: true
-    additional_functions: third_party/inspec/custom_functions/bucket_name_from_params.erb
```

Comment:
Does this file need to be deleted?

Reply:
It's used in a couple other resources in this file. Things that exist in storage buckets don't usually get the name of the bucket passed back in the response, but we want to have it on there